src/instructlab/eval/mmlu.py

```diff
@@ -30,10 +36,49 @@ def run(self) -> tuple:
             overall_score       MMLU score for the overall model evaluation
             individual_scores   Individual MMLU score for each task
         """
-        individual_scores: dict[str, float] = {}
-        overall_score: float = 0.0
+        #TODO: make this a parameter for class?
+        os.environ['TOKENIZERS_PARALLELISM'] = 'true'
```

Comment:
I think this could be a CLI config, either via flag or profile, @cdoern wdyt?

Reply:
@xukai92 if you know could you explain what TOKENIZERS_PARALLELISM is giving us? The reason I added this is because the output when I ran without setting that environment var was giving me warning to set that environment variable. 

For the sake of moving this PR along I say we just make a low priority issue for this TODO and someone can investigate this further.

Reply:
this should be in the eval profile, but for now will be a cli flag.

click has a way to read from env vars into flags too